### PR TITLE
Add dummy search method

### PR DESCRIPTION
**Pull request by MechCoder**

### CODE REVIEW

**Comment by MechCoder** [comment]
@glouppe ;)

File: skopt/dummy_search.py [hunk 1]
```diff
@@ -0,0 +1,66 @@
+import numpy as np
+
+from sklearn.utils import check_random_state
+
+from scipy.optimize import OptimizeResult
+
+def dummy_search(func, bounds=None, maxiter=1000, random_state=None):
```
Comment (glouppe) [inline skopt/dummy_search.py]: Can we try to enforce some consistency on naming? What do you think of `dummy_minimize` instead?

File: skopt/dummy_search.py [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+        http://docs.scipy.org/doc/scipy/reference/generated/scipy.optimize.OptimizeResult.html
+    """
+    rng = check_random_state(random_state)
+
+    # Bounds
+    num_params = len(bounds)
+    lower_bounds, upper_bounds = zip(*bounds)
+    upper_bounds = np.asarray(upper_bounds)
+    lower_bounds = np.asarray(lower_bounds)
+    diff = upper_bounds - lower_bounds
```
Comment (glouppe) [inline skopt/dummy_search.py]: Shall we factor out this piece of code in some `utils`? (This also appears in `gp_minimize`)

**Comment by glouppe** [comment]
Thanks for this! +1 once my comments are fixed

**Comment by MechCoder** [comment]
Will merge pending Travis..

**Comment by betatim** [comment]
:dart:

**Pull request merged**